modules/minimum-required-permissions-ipi-azure.adoc

```diff
@@ -235,12 +232,6 @@ The following permissions are not required to create the private {product-title}
 
 The following permissions are required for deleting an {product-title} cluster on {azure-full}. You can use the same permissions to delete a private {product-title} cluster on {azure-short}.
 
-.Required permissions for deleting authorization resources
-[%collapsible]
-====
-* `Microsoft.Authorization/roleAssignments/delete`
-====
```

Comment:
Similar to other comments, this is now needed when the installer creates an identity (default)